hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2687,17 +2687,15 @@ func reconcileCAPIManagerRole(role *rbacv1.Role) error {
 				"hostedcontrolplanes",
 				"hostedcontrolplanes/status",
 			},
-			Verbs: []string{"*"},
+			Verbs: []string{"get", "list", "watch"},
 		},
 		{
 			APIGroups: []string{""},
 			Resources: []string{
 				"configmaps",
-				"events",
-				"nodes",
 				"secrets",
 			},
-			Verbs: []string{"*"},
+			Verbs: []string{"list", "get", "watch"},
```

Comment:
picky nit: can you keep these in the same order for consistency?

Reply:
yes, added!